android/src/main/java/com/joinflux/flux/capacitorsegment/CapacitorSegmentPlugin.java

```diff
@@ -26,24 +26,24 @@ public void initialize(PluginCall call) {
             return;
         }
         String key = call.getString("key");
-        Boolean trackLifecycle = call.getBoolean("trackLifecycle", false);
         if (key == null) {
             call.reject("Write key is required to initialize plugin");
             return;
         }
 
         Context context = this.getContext();
-        this.getActivity().runOnUiThread(new Runnable() {
-            @Override
-            public void run() {
-                Builder builder = new Analytics.Builder(context, key);
-                if (trackLifecycle) {
-                    builder.trackApplicationLifecycleEvents();
-                }
-                initialized = true;
-                implementation.analytics = builder.build();
-            }
-        });
+        Builder builder = new Analytics.Builder(context, key);
+        Boolean trackLifecycle = call.getBoolean("trackLifecycle", false);
+        if (trackLifecycle) {
+            builder.trackApplicationLifecycleEvents();
+        }
+
+        Boolean recordScreenViews = call.getBoolean("recordScreenViews", false);
```

Comment:
I think we'll also need to update `src/definitions` to add a boolean for `recordScreenViews` for consumers via JS

Reply:
Ah good catch